app/core.go

```diff
@@ -1463,7 +1473,8 @@ changeLoop:
 	}
 
 	logger.Debug("global report complete", zap.Int("num-lines", len(lines)))
-	if _, err := a.chat.SendChannelReport(ctx, chanID, "Waiting for review", lines); err != nil {
+	caption := fmt.Sprintf("Waiting for review %d (%d over week, %d over month)", len(lines), countOverWeek, countOverMonth)
```

Comment:
Could we word it more like `"%d changesets waiting for review (%d waiting for over a week, %d waiting for over a month)"`? It wasn't obvious to me what the numbers meant until I read the code.

Reply:
Sounds good.